src/index.html

```diff
@@ -1,17 +1,219 @@
+              value="12"
+              min="12"
+              max="100"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="birthday">
+            Full date of birth:
+            <input
+              type="date"
+              name="birthday"
+              id="birthday"
+              min="1900-01-01"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="agreement">
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="agreement"
+              id="agreement"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="password">
+            Password:
+            <input
+              type="password"
+              name="password"
+              id="password"
+              minlength="5"
+              maxlength="30"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label for="first-animal">
+            <input
+              type="radio"
+              name="animals"
+              id="first-animal"
+              value="y"
+            >
+            Yes
+          </label>
+
+          <label for="second-animal">
+            <input
+              type="radio"
+              name="animals"
+              id="second-animal"
+              value="n"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="color">
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              id="color"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="time">
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time"
+              id="time"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="cars">
+            What are your favorite brand of cars?
+            <select
+              name="cars"
+              id="cars"
+              multiple
+            >
+              <option value="car">BMW</option>
+              <option value="car">Audi</option>
+              <option value="car">Lada</option>
```

Comment:
Value should describe exact choice, 
```suggestion
              <option value="bmw">BMW</option>
              <option value="audi">Audi</option>
              <option value="lada">Lada</option>
```